project-p4-chatbot-master/code/script.js

```diff
@@ -0,0 +1,293 @@
+      <option value="" selected disabled>👇Wähle einen Grund aus.</option>
+      <option value="Tolle Gelegenheit neue Talente zu entdecken.">Tolle Gelegenheit neue Talente zu entdecken.</option>
+      <option value="Freude, andere Menschen auf ihrem Weg zu unterstützen">Freude, andere Menschen auf ihrem Weg zu unterstützen.</option>
+      <option value="Möglichkeit, potentielle neue Mitarbeiter finden.">Möglichkeit, potentielle neue Mitarbeiter finden.</option>
+      <option value="Anderer Grund">Anderer Grund</option>
+    </select>
+  `
+  document.getElementById('select').addEventListener('change', handleCompanySelectInput);  
+}
+
+const handleJustInput = (reason) => {
+  console.log("just")
+  showMessage("Ich möchte mich nur mal umschauen.", 'user')
+
+  setTimeout(() => showJustOptions(), 1000)
+}
+
+const showJustOptions = (reason) => {
+  showMessage(`Schön, dass du dich auf unserer Webseite umschaust. Gefällt dir, was du bis jetzt entdeckt hast?`, 'bot')
+
+  inputWrapper.innerHTML = `
+  <select id="select"> 
+    <option value="" selected disabled>👇 Wähle das entsprechende Emoji aus.</option>
+    <option value="👍👍😍">👍👍😍</option>
+    <option value="👍">👍</option>
+    <option value="😐">😐</option>
+    <option value="👎e">👎</option>
+  </select>
+  `
+  document.getElementById('select').addEventListener('change', handleJustSelectInput);  
+}
+
+const handleJobsurferSelectInput = () => {
+  console.log("jobsurferselect")
+  showMessage(select.value, 'user')
+
+  setTimeout(() => ThankYouJobsurfer1(), 2000)
+}
+
+const ThankYouJobsurfer1 = () => {
+  showMessage("Ich bin mir sicher, dass der Einblick in andere Berufsfelder sehr bereichernd ist und dir Klarheit verschaffen wird.", 'bot')
+  setTimeout(() => ThankYouJobsurfer2(), 4000)
+}
+
+const ThankYouJobsurfer2 = () => {
+    showMessage("Uns von jobsurfing.ch liegt es am Herzen, dass Du einen Job findest, der dich erfüllt und glücklich macht. 💜", 'bot')
+  setTimeout(() => showYesOrNo(), 3000)
+}
+
+const handleCompanySelectInput = () => {
+  console.log("companyselect")
+  showMessage(select.value, 'user')
+
+  setTimeout(() => ThankYouCompany1(), 2000)
+}
+
+const ThankYouCompany1 = () => {
+  showMessage("Wie du siehst, profitiert auch das Unternehmen von Jobsurfing. Einige Gründe findest du auf unserer Startseite.", 'bot')
+  setTimeout(() => ThankYouCompany2(), 3000)
+}
+
+const ThankYouCompany2 = () => {
+  showMessage("Uns von jobsurfing.ch liegt es am Herzen, Jobsurfer und Unternehmen miteinander so zu vernetzen, damit die Jobsurfing Erfahrung für beide bereichernd ist. 💜", 'bot')
+setTimeout(() => showYesOrNo(), 4000)
+}
+
+const handleJustSelectInput = () => {
+  console.log("justselect")
+  showMessage(select.value, 'user')
+
+  setTimeout(() => ThankYouJust(), 2000)
+}
+
+const ThankYouJust= () => {
+  showMessage("Vielen Dank für deine Rückmeldung! Das hilft uns dabei, unsere Webseite zu optimieren.", 'user')
+  setTimeout(() => showYesOrNo(), 2000)
+}
```

Comment:
Also, it would be nice if you prevented the user from picking another alternative once it clicked an option. You can do this by emptying the input wrapper